### PR TITLE
handle copy in all profiler tables

### PR DESCRIPTION
**Pull request by alanrenmsft**
#5667

there are 2 tables in profiler, adding copy support for both of them.

### CODE REVIEW

File: src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as os from 'os';
+import { IClipboardService } from 'sql/platform/clipboard/common/clipboardService';
+
+export function HandleCopyRequest(clipboardService: IClipboardService, range: Slick.Range, getCellValue: (row, cell) => string): void {
```
Comment (kburtram) [inline src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts]: `HandleCopyRequest` should be camelCased like `handleCopyRequest`

File: src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	if (range) {
+		let results = '';
+		for (let i = range.fromRow; i <= range.toRow; i++) {
```
Comment (kburtram) [inline src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts]: if this loop can take a long time to complete we should show the busy cursor while it's executing.  we have this problem with the results grid.

File: src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts (hunk 1 above)
Comment (alanrenmsft) [reply src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts]: good catch, i had the file name camel cased but forgot the method name...

File: src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts (hunk 2 above)
Comment (alanrenmsft) [reply src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts]: Profiler might not have this problem, for the events table, it only copies the active cell, for the details table, the number of records are the same as the number of columns.

**Pull request merged**